app/src/main/java/org/wikipedia/language/LangLinksActivity.kt

```diff
@@ -0,0 +1,367 @@
+package org.wikipedia.language
+
+import android.content.Context
+import android.os.Bundle
+import android.view.*
+import android.widget.TextView
+import androidx.appcompat.view.ActionMode
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.disposables.CompositeDisposable
+import io.reactivex.rxjava3.schedulers.Schedulers
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.activity.BaseActivity
+import org.wikipedia.databinding.ActivityLanglinksBinding
+import org.wikipedia.dataclient.ServiceFactory
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.dataclient.mwapi.SiteMatrix
+import org.wikipedia.history.HistoryEntry
+import org.wikipedia.history.SearchActionModeCallback
+import org.wikipedia.page.PageActivity
+import org.wikipedia.page.PageTitle
+import org.wikipedia.settings.SiteInfoClient
+import org.wikipedia.util.DeviceUtil
+import org.wikipedia.util.StringUtil
+import org.wikipedia.util.log.L
+import org.wikipedia.views.ViewAnimations
+import java.util.*
+
+class LangLinksActivity : BaseActivity() {
+    private lateinit var binding: ActivityLanglinksBinding
+
+    private var languageEntries = mutableListOf<PageTitle>()
+    private lateinit var title: PageTitle
```

Comment:
Would it be better if move this `private lateinit var` to the top for a better reading experience?